packages/internals/src/cli/getSchema.ts

```diff
@@ -57,6 +57,8 @@ type PackageJsonLookupResult =
       }
     }
 
+export type SchemaPathFromConfig = { kind: 'single'; filenamePath: string } | { kind: 'multi'; folder: string }
```

Comment:
Did you copy the type from `@prisma/config` because you didn't want to add a dependency to its package?

Reply:
Not really the direct package dependency itself (internals already depends on config in the package.json), but to not let an "inner" module depend on a "sibling"/"higher" module moving forward. So the schema loading logic has its own interface that it exposes based on what it needs. Which "just happens" to align very well with the config format. ;)